contrib_bots/lib/DefineBot.py

```diff
@@ -0,0 +1,74 @@
+# See readme.md for instructions on running this code.
+import json
+import requests
+import ssl
+
+class DefineHandler(object):
+    '''
+    This plugin define a word that the user inputs. It
+    looks for messages starting with '@define'.
+    '''
+
+    def usage(self):
+        return '''
+            This plugin will allow users to define a word. Users should preface
+            messages with "@define".
+            '''
+
+    def triage_message(self, message):
+        # return True iff we want to (possibly) response to this message
+
+        original_content = message['content']
+
+        # This next line of code is defensive, as we
+        # never want to get into an infinite loop of posting follow
+        # ups for own follow ups!
+        is_define = original_content.startswith('@define')
+
+        return is_define
+
+    def _handle_definition(self, original_content):
+        error_message = 'Definition not available.'
```

Comment:
move `error_message` to be class attr. Its also constant - it means it should be uppercase.